frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/quickfix/CreateBranchQuickFix.java

```diff
@@ -0,0 +1,72 @@
+package com.virtuslab.gitmachete.frontend.file.quickfix;
+
+import static com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getNonHtmlString;
+
+import java.util.Collections;
+
+import com.intellij.codeInsight.intention.IntentionAction;
+import com.intellij.codeInspection.util.IntentionFamilyName;
+import com.intellij.codeInspection.util.IntentionName;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiFile;
+import com.intellij.util.IncorrectOperationException;
+import git4idea.branch.GitBrancher;
+import git4idea.repo.GitRepository;
+import io.vavr.control.Option;
+import lombok.experimental.ExtensionMethod;
+
+import com.virtuslab.gitmachete.frontend.file.MacheteFileUtils;
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle;
+import com.virtuslab.qual.guieffect.UIThreadUnsafe;
+
+@ExtensionMethod({GitMacheteBundle.class})
+public class CreateBranchQuickFix implements IntentionAction {
+
+  private final String branch;
+  private final String parentBranch;
+  private final PsiFile macheteFile;
+
+  public CreateBranchQuickFix(String processedBranchName, String parentBranchName, PsiFile processedFile) {
+    branch = processedBranchName;
+    parentBranch = parentBranchName;
+    macheteFile = processedFile;
+  }
+
+  @Override
+  public @IntentionName String getText() {
+    return getNonHtmlString("action.GitMachete.BaseSlideInBelowAction.dialog.create-new-branch.title")
```

Comment:
I'd add a separate property solely for the purpose of this quick fix:
1. quick fix intentions should be written in _Sentence case_, not _Title Case_
2. there should be no need to concat the strings this way, just pass `branch` as an argument to format